std/haxe/coro/continuations/BlockingContinuation.hx

```diff
@@ -6,7 +6,7 @@ class BlockingContinuation implements IContinuation<Any> {
 	final loop:EventLoop;
 
 	var running:Bool;
-	var result:Int;
+	var result:Any;
```

Comment:
The most confusing thing about this one is that we don't come across is on kt_coro already, which I think means that we never actually call `resume` on our BlockingContinuation.